### PR TITLE
mariadb: Add basic test

### PR DESCRIPTION
**Pull request by miska**
Maintainer: me
Compile tested: NA
Run tested: NA

Description:
Try to add basic test for MariaDB, mainly to get a feel how CI works. Test to be enhanced later, locally script works, but want to ensure that I got how the whole system works.

### CODE REVIEW

**Comment by miska** [comment]
Trying to figure out how new tests work, but in the output I don't see the output anywhere.
@aparcar Do you have some tips how to test the tests? :-)

**Comment by aparcar** [comment]
I'm on it https://github.com/openwrt/packages/pull/13543

**Comment by aparcar** [comment]
I updated the script but your test currently fails:
```
Testing package mariadb-client-base in version 10.4.13 from mariadb
Installing mariadb-client-base (10.4.13-3) to root...
Installing mariadb-common (1.0-2) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/i386_pentium4/packages/mariadb-common_1.0-2_i386_pentium4.ipk
Installing libatomic1 (8.4.0-2) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/x86/generic/packages/libatomic1_8.4.0-2_i386_pentium4.ipk
Installing libopenssl1.1 (1.1.1h-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/i386_pentium4/base/libopenssl1.1_1.1.1h-1_i386_pentium4.ipk
Installing libstdcpp6 (8.4.0-2) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/x86/generic/packages/libstdcpp6_8.4.0-2_i386_pentium4.ipk
Installing libedit (20191231-3.1-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/i386_pentium4/packages/libedit_20191231-3.1-1_i386_pentium4.ipk
Configuring mariadb-common.
Configuring libatomic1.
Configuring libopenssl1.1.
Configuring libstdcpp6.
Configuring libedit.
Configuring mariadb-client-base.
Use package specific test.sh
Can't find MariaDB binary
Test failed
```

File: utils/mariadb/test.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if which mysqld 2> /dev/null >&2; then
```
Comment (neheb) [inline utils/mariadb/test.sh:3]: surprised shellcheck doesn't complain here. command -v should be used instead of which.

**Comment by miska** [comment]
Thank you! I can deal with broken test once it's actually getting executed and I have something to test again :-) Will take a look at it later today.

**Comment by aparcar** [comment]
Thanks for testing!

**Comment by aparcar** [comment]
Feel free to patch the entrypoint.sh with the colour functions.

**Comment by miska** [comment]
Tried to add colors in #13564

**Comment by aparcar** [comment]
Please rebase this based on #13564 and I'm happy to merge this.

**Pull request merged**